src/test/groovy/org/prebid/server/functional/util/privacy/GppConsent.groovy

```diff
@@ -0,0 +1,30 @@
+package org.prebid.server.functional.util.privacy
+
+import com.iab.gpp.encoder.GppModel
+import com.iab.gpp.encoder.section.TcfEuV2
+
+class GppConsent implements ConsentString {
+
+    private final GppModel gppModel
+
+    GppConsent() {
+        gppModel = new GppModel()
+    }
+
+    GppConsent setFieldValue(String sectionName = TcfEuV2.NAME,
```

Comment:
You're defeating the whole purpose of building a facade over a library. If we decide to change the library along the way, you will be forced to rewrite all tests because you will be directly using that said library in there instead of our facade.

Reply:
Then I'll use directly `GppModel()` and then we'll rewrite this after some clarification.